src/SBML.jl

```diff
@@ -7,10 +7,10 @@ using Symbolics
 include("structs.jl")
 include("version.jl")
 
+include("utils.jl")
```

Comment:
I was kinda thinking to move the VPtr and other types to a separate types.jl already; there might be more coming. Let's keep that in mind.

Reply:
(other types = now basically Maybe)